src/AlgebraicGeometry/ToricVarieties/NormalToricVarieties/attributes.jl

```diff
@@ -114,24 +114,36 @@ julia> C = Oscar.positive_hull([1 0]);
 
 julia> antv = affine_normal_toric_variety(C);
 
-julia> set_coordinate_names(antv, ["u"])
+julia> set_coordinate_names(antv, [:u])
```

Comment:
Will `set_coordinate_names(antv, ["u"])` still function with your changes, i.e. strings instead of symbols? I wish that this old input (vector of strings) is supported. Extending the support to a vector of symbols is fine with me. Could you please extend the doctest to showcase both approaches/inputs?

Same for `set_coordinate_names_of_torus`.

Reply:
(Unless I am mistaken, the answer to my first question is "yes".)

Reply:
My bad.